heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -189,29 +189,10 @@ private SchedulerUtils() {
       Config runtime,
       int shardId,
       Map<ExecutorPort, String> ports) {
-    return getExecutorCommand(config, runtime, Integer.toString(shardId), ports);
-  }
-
-  /**
-   * Utils method to construct the command to start heron-executor
-   *
-   * @param config The static config
-   * @param runtime The runtime config
-   * @param shardId the executor/container index
-   * @param ports a map of ports to use where the key indicate the port type and the
-   * value is the port
-   * @return String[] representing the command to start heron-executor
-   */
-  public static String[] getExecutorCommand(
```

Comment:
Kubernetes needs to be able to send a string as the shard id

Reply:
will change back